### PR TITLE
Adds all functions

### PR DESCRIPTION
**Pull request by JanekJaniak**
I did it, but without filter or map in remove function.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -14,12 +16,32 @@ const items = [
...
 }
 
 const renderItems = () => {
     //prints list
-}
\ No newline at end of file
+    list.textContent='';
+    items.forEach((item,index) => {
+        const listItem=document.createElement('li');
+        listItem.innerHTML=items[index].value + '<button data-deleteButton>Delete</button>';
```
Comment (zodybek) [inline index.js]: wydaje mi się, że to można lepiej zapisać, żeby nie przepisywać HTML-a do JS... ale nie mam teraz pomysłu jak ;-)

**Review by zodybek — APPROVED** [review]
no piknie

File: index.js [hunk 2]
```diff
@@ -14,12 +16,32 @@ const items = [
...
+    const newItem = input.value;
+    if(newItem==='') return;
+    let item ={
+        value:newItem,
+        id:items.length+1,
+    }
+    items.push(item);
+    renderItems();
+    input.value='';
 }
```
Comment (jan-gajdowicz) [inline index.js]: All this function should do is push an item into `items` array. 
It should take `newItem` as an argument. 
`newItem` should be an object with `id` and `value`

File: index.js [hunk 3]
```diff
@@ -14,12 +16,32 @@ const items = [
...
 
-const removeItem = (itemId) => {
+const removeItem = (e) => {
     //filter items
+    e.target.parentNode.remove();
 }
```
Comment (jan-gajdowicz) [inline index.js]: All this function should do is filter `items` array

File: index.js [hunk 4]
```diff
@@ -14,12 +16,32 @@ const items = [
...
 
 const renderItems = () => {
     //prints list
-}
\ No newline at end of file
+    list.textContent='';
```
Comment (jan-gajdowicz) [inline index.js]: This would not work

File: index.js (hunk 1 above)
Comment (jan-gajdowicz) [inline index.js]: `items[index] === item`

Comment (jan-gajdowicz) [reply index.js]: Please use `appendNode` method

File: index.js [hunk 5]
```diff
@@ -14,12 +16,32 @@ const items = [
...
+    items.forEach((item,index) => {
+        const listItem=document.createElement('li');
+        listItem.innerHTML=items[index].value + '<button data-deleteButton>Delete</button>';
+        list.appendChild(listItem); 
+        listItem.querySelector('[data-deleteButton]').addEventListener('click', removeItem);
+        })
+    }
+
+renderItems();
+addButton.addEventListener('click', addItem);
```
Comment (jan-gajdowicz) [inline index.js]: Pass a `newItem` argument to `addItem` & call `renderItems`

**Review by jan-gajdowicz — CHANGES_REQUESTED** [review]

**Comment by jan-gajdowicz** [comment]
> no piknie

@zodybek Please try to avoid unrelated comments

**Comment by jan-gajdowicz** [comment]
@JanekJaniak Please try to implement code review feedback
@zodybek Please try commenting in english and do not approve a PR that is not ready to be merged

**Comment by JanekJaniak** [comment]
I did new render function. Don't know how and why I need to use filter method in remove function, so I stopped there.

**Pull request merged**